l10n_it_DDT_webkit/report/ddt.mako

```diff
@@ -115,7 +115,7 @@
                 <td><br /><br /></td>
             </tr>
         </table>
-         <p style="page-break-after: always"/>
+         <p style="page-break-after: never"/>
```

Comment:
In questo modo però sel'utente seleziona 2 pickings (mettiamo di una sola riga ciascuno) e lancia la stampa DDT, i 2 DDT vengono stampati in una sola pagina.

Per la soluzione completa bisognerebbe seguire ciò che fa invoice_webkit: https://github.com/OCA/account-invoice-reporting/blob/7.0/invoice_webkit/report/account_invoice.mako#L396